### PR TITLE
5/11 perf meeting updates

### PR DESCRIPTION
**Pull request by psahabu**
@chapel-lang/perf-team:
- Added annotations for #6212:
  - PRK stencil graphs
  - NPB
- added y-axis to array init tests
- removed empty task spawning from perf tracking
- PRK stencils were already part of 16-node-xc perf tracking

Having trouble running `check_annotations.py`, so I'm holding off until Monday.

### CODE REVIEW

**Comment by ronawho** [comment]
What does "Having trouble running check_annotations.py" mean? If it's giving you any output beyond "No fatal annotation errors detected", that probably means you have a badly formatted annotation

File: test/ANNOTATIONS.yaml [hunk 1]
```diff
@@ -44,6 +44,8 @@ all:
   03/28/14:
     - Add static flag for performance testing (22992)
   05/08/14:
+  05/11/17:
```
Comment (ben-albrecht) [inline test/ANNOTATIONS.yaml]: This may be the cause of the `check_annotations.py` failure

**Comment by psahabu** [comment]
@ronawho Yup that was the case, I just had absolutely no recollection of pasting the lines that @ben-albrecht pointed out so I was very confused by the error.

**Comment by ben-albrecht** [comment]
@psahabu -- it doesn't look like you pushed your fix before merging?

**Comment by psahabu** [comment]
@ben-albrecht:
Wow this is a rough Monday, I can't believe I overlooked that. On it.

**Pull request merged**